Polyhedron/demo/Polyhedron/Viewer.h

```diff
@@ -174,6 +175,7 @@ public Q_SLOTS:
   QString message;
   bool _displayMessage;
   QTimer messageTimer;
+  QMatrix4x4 projectionMatrix;
```

Comment:
See commit https://github.com/CGAL/cgal/pull/1244/commits/5d1da468bf758415d7b6f6124c605a040683dd48. There was already a d-pointer for `Viewer`.